pkg/keycloak/proxy/server.go

```diff
@@ -1006,6 +1006,14 @@ func (r *OauthProxy) Run() error {
 	return nil
 }
 
+// Shutdown finishes the proxy service with gracefully period
+func (r *OauthProxy) Shutdown() {
+	gracefulPeriod := r.Config.ServerGraceTimeout
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(time.Millisecond*gracefulPeriod))
```

Comment:
hmm gracefulPeriod is in seconds why we are multiplying it with miliseconds?

Reply:
I was following the other durations. Set as Seconds